preprocessing/data_preprocessor.py

```diff
@@ -122,12 +102,91 @@ def segment_nifti_files(self, nifti_file_path: str) -> None:
         """
         out_path = os.path.join(self.segment_dir, nifti_file_path.split(
             '/')[-1].replace('.nii.gz', ''))
-        totalsegmentator(nifti_file_path, out_path, roi_subset=[
-                         'liver', 'spleen', 'kidney_right', 'kidney_left', 'small_bowel'])
+        totalsegmentator(nifti_file_path, out_path, roi_subset=ORGANS)
+
+    def json_organs(self, subject) -> None:
```

Comment:
I supose this function is the one cropping the organ segmentations, but I miss a docstring here explaining what exacly the function is suposed to do. @FrancescVP

Reply:
TODO added